### PR TITLE
feat: Privacy policy

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: proto/zitadel/admin.proto [hunk 1]
```diff
@@ -1450,6 +1450,65 @@ service AdminService {
...
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            tags: "policy";
+            tags: "privacy policy";
+            tags: "privacy";
+            responses: {
+                key: "200";
+                value: {
+                    description: "default password complexity policy";
```
Comment (livio-a) [inline proto/zitadel/admin.proto:1470]:
```suggestion
                    description: "default privacy policy";
```

File: proto/zitadel/management.proto [hunk 2]
```diff
@@ -1927,6 +1927,70 @@ service ManagementService {
...
+            get: "/policies/privacy"
+        };
+
+        option (zitadel.v1.auth_option) = {
+            permission: "policy.read"
+        };
+    }
+
+    // Returns the default privacy policy of the IAM
+    // With this policy the password strength can be configured
```
Comment (livio-a) [inline proto/zitadel/management.proto:1943]:
```suggestion
    // With this policy privacy relevant things can be configured (e.g. tos link)
```

File: internal/command/setup_step17.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+func (c *Commands) SetupStep17(ctx context.Context, step *Step17) error {
+	fn := func(iam *IAMWriteModel) ([]eventstore.EventPusher, error) {
+		iamAgg := IAMAggregateFromWriteModel(&iam.WriteModel)
+		addedPolicy := NewIAMPrivacyPolicyWriteModel()
+		events, err := c.addDefaultPrivacyPolicy(ctx, iamAgg, addedPolicy, &step.PrivacyPolicy)
+		if err != nil {
+			return nil, err
+		}
+
+		logging.Log("SETUP-4k0LL").Info("default message text set up")
```
Comment (livio-a) [inline internal/command/setup_step17.go:31]:
```suggestion
		logging.Log("SETUP-Ghbr3").Info("default privacy policy set up")
```

File: internal/iam/repository/view/model/privacy_policy.go [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+type PrivacyPolicyView struct {
+	AggregateID  string    `json:"-" gorm:"column:aggregate_id;primary_key"`
+	CreationDate time.Time `json:"-" gorm:"column:creation_date"`
+	ChangeDate   time.Time `json:"-" gorm:"column:change_date"`
+	State        int32     `json:"-" gorm:"column:state"`
+
+	TOSLink     string `json:"tosLink" gorm:"column:tos_link"`
+	PrivacyLink string `json:"privacyLink" gorm:"column:privacy_link"`
+	Default     bool   `json:"-" gorm:"-"`
```
Comment (livio-a) [inline internal/iam/repository/view/model/privacy_policy.go:31]: won't default always be false... it is never set to true

File: internal/ui/login/static/i18n/de.yaml [hunk 5]
```diff
@@ -160,9 +160,7 @@ Registration:
   TosConfirm: Ich akzeptiere die
   TosLinkText: AGBs
   TosConfirmAnd: und die
-  TosLink: https://docs.zitadel.ch/docs/legal/terms-of-service
```
Comment (livio-a) [inline internal/ui/login/static/i18n/de.yaml:163]: shouldn't there be the same changes for en.yaml?

File: internal/renderer/renderer.go [hunk 6]
```diff
@@ -15,6 +15,8 @@ import (
 
 const (
 	TranslateFn = "t"
+	TosFunc     = "hasTosLink"
+	PrivacyFunc = "hasPrivacyLink"
```
Comment (livio-a) [inline internal/renderer/renderer.go:19]: as far as i can see these funcs are never used

File: internal/ui/login/handler/renderer.go [hunk 7]
```diff
@@ -184,6 +184,12 @@ func CreateRenderer(pathPrefix string, staticDir http.FileSystem, staticStorage
 		"selectedGender": func(g int32) bool {
 			return false
 		},
+		"hasTOSLink": func() bool {
+			return false
+		},
+		"hasPrivacyLink": func() bool {
+			return false
+		},
```
Comment (livio-a) [inline internal/ui/login/handler/renderer.go:192]: as far as i can see these funcs are never used

File: internal/ui/login/static/templates/register.html [hunk 8]
```diff
@@ -91,23 +91,31 @@ <h1>{{t "Registration.Title"}}</h1>
...
+                        </a>
+                    {{end}}
+                    {{ if and .TOSLink .PrivacyLink }}
+                        {{t "Registration.TosConfirmAnd"}}
+                    {{ end }}
+                    {{ if .PrivacyLink }}
+                        <a class="tos-link" target="_blank" href="{{ .PrivacyLink}}" rel="noopener noreferrer">
+                        {{t "Registration.PrivacyLinkText"}}
+                        </a>
+                    {{end}}
```
Comment (livio-a) [inline internal/ui/login/static/templates/register.html:114]: did you try all 3 combinations? just wondered if spacing (between words) is really correct

**Review by livio-a — CHANGES_REQUESTED** [review]
please rebuild and push api docs after changes

File: internal/ui/login/static/templates/register.html (hunk 8 above)
Comment (fgerschwiler) [reply internal/ui/login/static/templates/register.html]: yes mow it works

**Review by livio-a — APPROVED** [review]

**Pull request merged**